Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -74,6 +67,8 @@ void Digitizer::init()
     mDriftEstimators.emplace_back();
   }
 
+  std::fill(mFlagVdriftOutOfRange.begin(), mFlagVdriftOutOfRange.end(), false);
+
```

Comment:
I think this isn't needed since `mFlagVdriftOutOfRange` is automatically initialized to its default value since it's a `std::array` of `bools`, see similar comment in the header.

Reply:
Yes, changed